SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitPaymentAuthRequestParams.java

```diff
@@ -0,0 +1,25 @@
+package com.braintreepayments.api;
+
+import androidx.fragment.app.FragmentActivity;
+
+/**
+ * Returned via the {@link SEPADirectDebitPaymentAuthRequestCallback} after calling
+ * {@link SEPADirectDebitClient#createPaymentAuthRequest(SEPADirectDebitRequest, SEPADirectDebitPaymentAuthRequestCallback)}.
+ *
+ * Inspect the {@link SEPADirectDebitNonce} property to determine if tokenization is complete, or
+ * if you must continue the SEPA mandate web flow via
+ * {@link SEPADirectDebitLauncher#launch(FragmentActivity, SEPADirectDebitPaymentAuthRequest.ReadyToLaunch)}
+ */
+public class SEPADirectDebitPaymentAuthRequestParams {
```

Comment:
QQ: I see that most of the newly created files are in kotlin. Any reason this is an exception?

Reply:
Ah this file isn't actually new it was renamed from the previous SEPADirectDebitPaymentAuthRequest class (which is replaced by the new Kotlin sealed class in this PR). We are slowly migrating to Kotlin, so have been trying to add new files in Kotlin and convert small interfaces, but I didn't refactor this one into Kotlin yet.